### DISCUSSION TITLE
Can you merge cells?

### DISCUSSION THREAD

**Discussion by Bromson3105** [Akascape/CTkTable#87] [Q&A]
Is there any way to merge cells?
I would like to create a table that has the format:
![image](https://github.com/Akascape/CTkTable/assets/64321583/da771aa5-7f43-48c4-bf12-0d772e31feca)

Right now, the header is actually not part of the table but rather a CTkLabel on the same CTkFrame without padding

Alternatively, can I create an "empty" table with "zero rows and two columns"?
Meaning, I´m adding the rows iteratively from a dictionary, so I first need to create the table and then add the rows one by one. But if I try to do this, I get first an empty row and then my rows (right now "George 35" is added manually as starting values for the table), like this:
![image](https://github.com/Akascape/CTkTable/assets/64321583/e9ef831a-9733-4a77-b07c-eb12e8e0805a)

**Comment by Akascape** [comment]
@Bromson3105 This is a logical implementation you will need to do. There is no built in methods in ctktable.
